utils/goutils_test.go

```diff
@@ -82,16 +82,24 @@ func TestGetProjectRoot(t *testing.T) {
 }
 
 func TestGetDependenciesList(t *testing.T) {
-	testGetDependenciesList(t, "testGoList")
+	testGetDependenciesList(t, "testGoList", nil)
 }
 
 func TestGetDependenciesListWithIgnoreErrors(t *testing.T) {
 	// In some cases, we see that running go list on some Go packages may fail.
 	// We should allow ignoring the errors in such cases and build the Go dependency tree, even if partial.
-	testGetDependenciesList(t, "testBadGoList")
+	testGetDependenciesList(t, "testBadGoList", nil)
+	// In some cases we would like to exit after we receive an error, this can be done with custom error handle func.
+	// this test handleErrorFunc return an error
```

Comment:
// In some cases we would like to exit after we receive an error, this can be done with custom error handle func.
// this test handleErrorFunc return an error
-->
// In some cases we would like to exit after we receive an error. This can be done with custom error handle func.
// This test handleErrorFunc return an error.

(We should avoid forcing two sentencing into one).